client/state/selectors/should-load-gutenframe.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Internal dependencies
+ */
+import { isEligibleForGutenframe } from 'calypso/state/gutenberg-iframe-eligible/is-eligible-for-gutenframe';
+import { shouldCalypsoifyJetpack } from 'calypso/state/selectors/should-calypsoify-jetpack';
+import isNavUnificationEnabled from 'calypso/state/selectors/is-nav-unification-enabled';
+import getUserSettings from 'calypso/state/selectors/get-user-settings';
+
+export const shouldLoadGutenframe = ( state, siteId ) => {
+	return (
+		isEligibleForGutenframe( state, siteId ) &&
+		shouldCalypsoifyJetpack( state, siteId ) &&
```

Comment:
Nitpick: `isEligibleForGutenframe` already checks `shouldCalypsoifyJetpack`, so do we need to check it again here?

Reply:
Yes, that's true. I think I must have forgotten we had it in there. I've updated it.